src/serial/MidiSessionALSA.cc

```diff
@@ -1,10 +1,18 @@
 #include "MidiSessionALSA.hh"
 #include "CliComm.hh"
 #include "MidiOutDevice.hh"
+#include "MidiInDevice.hh"
+#include "MidiInConnector.hh"
+#include "EventListener.hh"
+#include "EventDistributor.hh"
 #include "PlugException.hh"
+#include "Poller.hh"
 #include "PluggingController.hh"
+#include "Scheduler.hh"
 #include "narrow.hh"
 #include "serialize.hh"
+#include "circular_buffer.hh"
+#include "checked_cast.hh"
 #include <iostream>
 #include <memory>
```

Comment:
There's a missing `#include <thread>` here. That'll probably fix the compilation errors in the buildloop.